### PR TITLE
Update lang_es.php

### PR DESCRIPTION
**Pull request by heticor915**
Fixed  typos in the spanish language.

### CODE REVIEW

File: src/language/lang_es.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
   'Play' =>
     'Jugar',
   'Welcome to the Facebook Capture the Flag Competition. By clicking "Play," you will be entered into the official CTF challenge. Good luck in your conquest.' =>
-    'Bienvenido a la competición CTF de Facebook. Al hacer clic en "Jugar," entrarás en la competición CTF. ¡Mucha suerte!.',
+    'Bienvenido a la competencia CTF de Facebook. Al hacer clic en "Jugar," entrarás en la competencia CTF. ¡Mucha suerte!.',
```
Comment (javuto) [inline src/language/lang_es.php]: I prefer to use "competición"

File: src/language/lang_es.php [hunk 2]
```diff
@@ -66,7 +66,7 @@
...
-    'Al participar en el concurso, usted acepta liberar a Facebook y sus empleados, y a la organizadora del evento de cualquier responsabilidad, reclamos o acciones de cualquier tipo por lesiones, daños o pérdidas a las personas y bienes que puedan ir en relación con el concurso. Usted reconoce y acepta que Facebook y otros no se hacen responsables de fallos técnicos, de hardware o software, u otros errores o problemas que pueden ocurrir relacionados al concurso',
+    'Al participar en la competencia, usted acepta liberar a Facebook y sus empleados, y a la organizadora del evento de cualquier responsabilidad, reclamo(s) o accion(es) de cualquier tipo por lesiones, daños o pérdidas a las personas y bienes que puedan ir en relación con el concurso. Usted reconoce y acepta que Facebook y otros no se hacen responsables de fallos técnicos, de hardware o software, u otros errores o problemas que pueden ocurrir relacionados al concurso',
```
Comment (javuto) [inline src/language/lang_es.php]: Again, I prefer to use "competición".
Regarding "reclamos" and "acciones", I don't see the reason to change them.

File: src/language/lang_es.php [hunk 3]
```diff
@@ -48,7 +48,7 @@
   'Attacking Scoreboard' =>
     'Puntuaciones de Ataque',
   'No attacking the competition infrastructure. If bugs or vulns are found, please alert the competition organizers immediately.' =>
-    'Prohibido atacar la infraestructura de la competencia. Si se encuentra alguna vulnerabilidad, por favor alerten inmediatamente a los organizadores.',
+    'Prohibido atacar la infraestructura de la competencia. Si se encuentra alguna vulnerabilidad o falla, por favor alerten inmediatamente a los organizadores.',
```
Comment (javuto) [inline src/language/lang_es.php]: I would prefer to use here "o fallo".

File: src/language/lang_es.php (hunk 1 above)
Comment (heticor915) [inline src/language/lang_es.php]: Competencia is more general, and it goes with the RAE rules for general i18n

File: src/language/lang_es.php (hunk 2 above)
Comment (heticor915) [inline src/language/lang_es.php]: @javuto They're synonyms, but is what you want.

File: src/language/lang_es.php (hunk 1 above)
Comment (m1guelpiedrafita) [inline src/language/lang_es.php]: (I am spanish) Competición is rigth. Competencia means another thing.
Competición: People figthing to do something and win.
Competencia: Situation of companies engaged in a market offering or demanding the same product or service. (RAE)

File: src/language/lang_es.php (hunk 2 above)
Comment (m1guelpiedrafita) [inline src/language/lang_es.php]: Same that above

**Pull request closed without merging**